### PR TITLE
challenge pseudo-park

### PR DESCRIPTION
**Pull request by AntoineFachez**
You review that code? Me throw you a 🦒.

### CODE REVIEW

File: css-basics/css-basics_pseudo-park/css/styles.css [hunk 1]
```diff
@@ -22,13 +22,15 @@
...
-    The animals are very small.
-    The size of the emojis is controlled via the font size.
-    Set the font size to 32px.
+  The animals are very small.
+  The size of the emojis is controlled via the font size.
+  Set the font size to 32px.
   */
 }
+.animal {
+  font-size: 2rem;
```
Comment (adrianricken) [inline css-basics/css-basics_pseudo-park/css/styles.css:32]: font-size 2rem ?

File: css-basics/css-basics_pseudo-park/css/styles.css [hunk 2]
```diff
@@ -22,13 +22,15 @@
```
Comment (adrianricken) [inline css-basics/css-basics_pseudo-park/css/styles.css:22]: i think we should change background-color to "yellow"

File: css-basics/css-basics_pseudo-park/css/styles.css (hunk 1 above)
Comment (AntoineFachez) [reply css-basics/css-basics_pseudo-park/css/styles.css:32]: thx Adrian. here you go: 

🦒

Comment (AntoineFachez) [reply css-basics/css-basics_pseudo-park/css/styles.css:32]: 2rem equals 16px. You can redefine a rem in your css. Default size is 8px. rem is easier to handle especially when useing font-size.

**Pull request merged**